sysvinit/netbsd/cloudinitlocal

```diff
@@ -11,6 +11,9 @@ name="cloudinitlocal"
 start_cmd="start_cloud_init_local"
 start_cloud_init_local()
 {
+    test -e /etc/cloud/cloud-init.disabled \
+      && warn "cloud-init disabled by cloud-init.disabled file" \
```

Comment:
Is there a reason to warn on this file but not the other 3 netbsd files?

Reply:
In my mind once per boot was enough to warn that cloud-init is disabled. I guess since this PR doesn't standardize on that I can add it into the other 3.

Reply:
fixed (both netbsd and freebsd)